### DISCUSSION TITLE
Unable to connect to printer

### DISCUSSION THREAD

**Discussion by maxxcz69** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1790] [General]
Hello, 
i done everything, updated firmware, try all ports and baudrates but my TFT 35 still show No printer attached. Im using TFT conector only. I got set in marlin 2 ports, -1 and 0, also REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER but still it doesnt work. Is my tft broken?

**Comment by adcallow** [comment]
Same EXACT thing is happening to me. Did EVERYTHING. Can't tell you how much time was wasted building and research different message boards and youtube videos. Good luck on this and I'll be monitoring as well should anyone have a suggestion outside of the normal "try switching the baud/port/FULL_GRAPHIC" comments that always show up.

**Comment by maxxcz69** [comment]
In the end i did it. I set ports to 0 and -1 and baudrato to 250000... but im unable to force ESP-01 to conncet to printer... when i set port to 3, then the FTF dont connect to printer again... glitchy...

**Comment by oldman4U** [comment]
For all users reading this discussion.

Please read the Readme at the front page of this repository. There you find everything related to connecting the TFT.

No printer attached: Correct serial port defined in Marlin (mainbord dependent) and matching baudrate.

Marlin Emulation is empty: correct display type defined in Marlin.
